File: app/src/main/java/com/example/markkko/povezime/app/util/NumberUtils.java
package com.example.markkko.povezime.app.util;

public class NumberUtils {

  public static float clean(Float number) {
    if (number == null) return 0;
    return number;
  }

  public static int clean(Integer number) {
    if (number == null) return 0;
    return number;
  }

  public static long clean(Long number) {
    if (number == null) return 0L;
    return number;
  }

  public static double clean(Double number) {
    if (number == null) return 0;
    return number;
  }
}

File: app/src/main/java/com/example/markkko/povezime/app/util/NumberUtils.kt
package com.example.markkko.povezime.app.util

fun clean(number: Float?): Float {
    return number ?: 0f
}

fun clean(number: Int?): Int {
    return number ?: 0
}

fun clean(number: Long?): Long {
    return number ?: 0L
}

fun clean(number: Double?): Double {
    return number ?: 0.0
}
